pom.xml

```diff
@@ -73,6 +74,9 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+<!--          <systemPropertyVariables>-->
```

Comment:
this needed?

Reply:
it's just a comment and it has been useful for debugging why jetty didn't start but as I didn't commit the jul.properties.
does it really hurt to a have a comment in a pom.xml?

Reply:
you did commit it 😄.

Is it useful enough to always have on?

what about
```suggestion
```suggestion
<!-- Uncomment to enable extra logging for debugging -->
<!--          <systemPropertyVariables>-->
```
```

Reply:
done